srcpkgs/draco/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'draco'
+pkgname=draco
+version=1.5.6
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_POSITION_INDEPENDENT_CODE:BOOL=true"
+hostmakedepends="pkg-config python3"
+short_desc="Library to compress/decompress 3D geometric meshes, point clouds"
+maintainer="Arjan Mossel <arjanmossel@gmail.com>"
+license="Apache-2.0"
+homepage="https://google.github.io/draco/"
+distfiles="https://github.com/google/draco/archive/refs/tags/${version}.tar.gz"
+checksum=0280888e5b8e4c4fb93bf40e65e4e8a1ba316a0456f308164fb5c2b2b0c282d6
+
+draco-devel_package() {
+	depends="${sourcepkg}>=${version}_${revision}"
+	short_desc+=" - development files"
+	pkg_install() {
+		vmove usr/include
+		vmove usr/lib/pkgconfig
+		vmove "usr/lib/*.a"
```

Comment:
why only the static lib?

Reply:
The package has no `.so`s

Reply:
on both of these packages with only .as, is there no configure option to enable shared libraries?

Reply:
Oh you're right, the generic cmake option works.